product-features/token-swaps/key-concepts.md

```diff
@@ -0,0 +1,46 @@
+---
+description: >-
+  Learn the key terms involved in ALEX swap operations.
+---
+
+# Key concepts
+
+### Token Swap
+
+A Token Swap is the exchange or trade of a certain amount of one crypto asset for another. In ALEX Lab Platform, swaps are performed on the ALEX's decentralized exchange (DEX) and are facilitated by liquidity pools.
+
+### Swap Transaction
+
+The ALEX DEX operates through smart contracts built on the Stacks blockchain, so all interactions, including swaps, are carried out through blockchain transactions. When you confirm a swap transaction on the ALEX Lab Platform, you are submitting it to the Stacks network to interact with the ALEX DEX smart contract. Once the transaction is confirmed, it means the swap was executed successfully. If the transaction is reverted, no funds will be lost.
+
+### Base Token
+
+The token you currently hold and want to exchange. Is the token you will transfer to ALEX smart contract during the swap transaction.
+
+### Target Token
+
+Also known as the "quoted token", this is the token you will receive in the swap transaction in exchange for the base token.
+
+### Exchange Rate
+
+The exchange rate determines how many target tokens you would receive for one base token. On ALEX DEX, this rate is algorithmically determined by the [ALEX Automated Market Maker (AMM)](../../detailed-information/alexs-automated-market-maker-amm.md) protocol and is updated after each swap.
+
+### Swap Fee
+
+Is the cost associated with performing a swap. It is deducted from the base token amount and is tipically set at 0.5%, though it can vary depending on the token pair (liquidity pool) involved.
```

Comment:
This is the cost associated with performing a swap. It is deducted from the base token amount and is typically set at 0.5%, though it can vary depending on the token pair (liquidity pool) involved.

Reply:
Done.